### DISCUSSION TITLE
[Script request] Baikal

### DISCUSSION THREAD

**Discussion by kiwinh0** [community-scripts/ProxmoxVE#168] [Request script]
### Application Name

Baikal

### Website

https://sabre.io/baikal/install/

### Description

Baïkal is a lightweight CalDAV+CardDAV server. It offers an extensive web interface with easy management of users, address books and calendars. It is fast and simple to install and only needs a basic php capable server. The data can be stored in a MySQL or a SQLite database.

Baïkal allows to seamlessly access your contacts and calendars from every device. It is compatible with iOS, Mac OS X, DAVx5 on Android, Mozilla Thunderbird and every other CalDAV and CardDAV capable application. Protect your privacy by hosting calendars and contacts yourself - with Baïkal.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by networkr0** [comment]
I tried to run this script on got these errors:

`[ERROR] in line 159: exit code 0: while executing command "$@" > /dev/null 2>&1
/dev/stdin: line 2500: post_update_to_api: command not found
/dev/stdin: line 956: pop_var_context: head of shell_variables not a function context
/dev/stdin: line 18: pop_var_context: head of shell_variables not a function context

[ERROR] in line 1344: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(cat "$(dirname "$CORE_DIR")/install/${var_install}.sh")"

[10:46:57 PM] ✅ SSH script execution finished with code: 127
`
